src/main/kotlin/no/nav/syfo/client/isbehandlerdialog/BehandlerdialogClient.kt

```diff
@@ -0,0 +1,37 @@
+package no.nav.syfo.client.isbehandlerdialog
+
+import io.ktor.client.plugins.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import no.nav.syfo.client.azuread.v2.AzureAdV2Client
+import no.nav.syfo.client.httpClient
+import no.nav.syfo.util.bearerHeader
+
+class BehandlerdialogClient(
```

Comment:
`BehandlerdialogClient` eller `IsBehandlerdialogClient`? Har droppet `is` flere steder, vet ikke hva som leser best.

Reply:
Is-prefix på klasse-navn leser rart, så synes BehandlerdialogClient funker bra :+1: